apps/server/prisma/seed.ts

```diff
@@ -23,13 +23,13 @@ const MANAGER_SIG_FIELD_UUID = '1727a4d2-b22c-42de-b63d-5f553e964d75';
 const DIR_SIG_FIELD_UUID = '6bd5c08b-f309-4226-8914-7fef4ba631c2';
 
 // type definition for employee data used in upsertEmployee
-type EmployeeData = {
-  id: string;
-  firstName: string;
-  lastName: string;
-  email: string;
-  positionId: string;
-};
+// type EmployeeData = {
```

Comment:
we're not using this type?

Reply:
it was giving me a prettier error so I commented it out while trying to fix something in seed